modules/tinymce/src/core/main/ts/api/html/Styles.ts

```diff
@@ -265,6 +269,11 @@ const Styles = (settings: StylesSettings = {}, schema?: Schema): Styles => {
               value = value.toLowerCase();
             }
 
+            // Convert transparent to fully transparent black
+            if (value === 'transparent') {
+              value = '#00000000';
+            }
```

Comment:
But this isn't what we did in TinyMCE 5, we left it as `transparent`.

Can we not just leave it? The conversion to hex is only supposed to apply to `rgb()` colors, not named colors.

Reply:
Yeah, sorry had some confusing yesterday